docs/pages/docs/_meta.json

```diff
@@ -1,13 +1,13 @@
 {
   "index": "Introduction",
-  "guide": "Guide",
+  "guide": "",
   "-- Themes --": {
     "type": "separator",
     "title": "Themes"
   },
-  "docs-theme": "Docs Theme",
-  "blog-theme": "Blog Theme",
-  "custom-theme": "Custom Theme",
+  "docs-theme": "",
+  "blog-theme": "",
+  "custom-theme": "",
```

Comment:
I propose to capitalize undefined files in `_meta.json` and files with empty titles because in 99.9% cases, you don't want to render empty titles, even if you'll want you can use `" "`

in the above case we DRY and don't repeat `Docs Theme`, `Blog Theme`, etc. this will be automatically handled by `title` package